byzcoin/collect_tx.go

```diff
@@ -165,6 +177,10 @@ func (p *CollectTxProtocol) Dispatch() error {
 			case <-p.closing:
 				return nil
 			}
+
+			if vb.hasThresholdFor(leaderVersion) {
+				p.CommonVersionChan <- leaderVersion
+			}
```

Comment:
why is this part inside the for loop? shouldn't it be outside because it's waiting for all the responses and then computing the common version from those responses?

Reply:
if it's outside the channel size can be 1